src/field_theory/cardinality.lean

```diff
@@ -38,6 +38,7 @@ lemma fintype.is_prime_pow_card_of_field {α} [fintype α] [field α] : is_prime
 begin
   casesI char_p.exists α with p _,
   haveI hp := fact.mk (char_p.char_is_prime α p),
+  letI : algebra (zmod p) α := zmod.algebra _ _,
```

Comment:
Out of scope for this PR but we probably want to make an `algebra (zmod p)` version of `char_p.exists`:
```suggestion
  -- TODO: `algebra` version of `char_p.exists`, of type `Σ p, algebra (zmod p) α`
  casesI char_p.exists α with p _,
  haveI hp := fact.mk (char_p.char_is_prime α p),
  letI : algebra (zmod p) α := zmod.algebra _ _,
```